### DISCUSSION TITLE
Combine LS-Dyna and Native Radioss input decks for Starter input.

### DISCUSSION THREAD

**Discussion by DavidR-Vortex** [OpenRadioss/OpenRadioss#902] [Q&A] [answered]
Hello,

Is there a way to achieve this? Reason I ask is not all LS-Dyna functions are mapped but exist natively, *TERMINATION for sensors etc & we would like to maintain compatibility with our current pre-processing codes as much as possible.

Many thanks,
David

**Comment by MarianBulla** [comment, marked as answer]
Hello,
thank you for bringing up this point. 
Yes, we are working on that. So stay tuned :-)
Thanks and best regards,
Marian

**Reply by DavidR-Vortex** [reply]
That's fantastic news, thank-you.
